src/ringmesh/geogram_extension/geogram_extension.cpp

```diff
@@ -604,9 +604,11 @@ namespace
     private:
         vec3 load_vertex( GEO::LineInput& file, index_t field ) const
         {
-            double x = file.field_as_double( field++ );
-            double y = file.field_as_double( field++ );
-            double z = file.field_as_double( field++ );
+            double x = file.field_as_double( field );
+            field++;
```

Comment:
I have always this question about the pre- and post- incrementation. For native type, there is no difference between i++ and ++i. But for more complex object, there is a difference since a copy is make for object_of_complex_type++ but not for ++object_of_complex_type. So why do we not homogenize incrementation by always using pre-incrementation?

Reply:
Because it depends what you want to do.... http://cpp.sh/3phc
However, on a single line, this is the same. https://godbolt.org/g/Fba29P

Reply:
I totally agree that there is a difference if ++ is used inside a function call. But since we aim at remove this kind of code smell, it will not occur.